data/IPF.txt

```diff
@@ -0,0 +1,17 @@
+trait	#chrom	pos	ref	alt	beta	se	pval	study_doi	rsid
+IPF (meta-analysis)	3	44860894	A	T	0.457424847038875	0.0749444488849256	5.12e-10	10.1164/rccm.201905-1017OC	rs78238620
```

Comment:
the effect allele I believe should be A,..... they say that MINOR is effect allele so need to check which is minor